_posts/2020-05-08-new-release-2020-2.md

```diff
@@ -0,0 +1,8 @@
+---
+title:  "Tableau 2020.2 released, Connector SDK now targets that version"
+abstract: "The Connector SDK for 2020.1 has released for those wishing to target that version. The main SDK branch now targets Tableau 2020.2"
```

Comment:
"The main SDK branch now targets Tableau 2020.2. The Connector SDK for 2020.1 has released for those wishing to target that version."

Maybe just changing the order will help clarify 2020.2 is out and this is what to do to target older releases.

Reply:
Good idea, swapped them.